SickSangHae/ViewModel/CoreDataViewModel.swift

```diff
@@ -59,11 +106,75 @@ extension CoreDataViewModel {
         getAllReceiptData()
     }
     
-    func updateReceiptStateData(target: Receipt, state: Status) {
+    func updateStatus(target: Receipt, to status: Status) {
+        switch status {
+        case .shortTermUnEaten:
+            updateStatusToUnEaten(target: target, to: status)
+        case .shortTermPinned:
+            updateStatusToUnEaten(target: target, to: status)
+        case .longTermUnEaten:
+            updateStatusToUnEaten(target: target, to: status)
+        case .Eaten:
+            updateStatusToEaten(target: target)
+        case .Spoiled:
+            updateStatusToSpoiled(target: target)
```

Comment:
```suggestion
        case .shortTermUnEaten, . shortTermPinned, . longTermUnEaten:
            updateStatusToUnEaten(target: target, to: status)
        case .Eaten, .Spoiled:
            updateStatusToEaten(target: target)
```

겹치는 케이스들은 함께 핸들링해도 될거가탸요 ㅎㅎ